sass/sass_wrapper.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * A Sass compiler wrapper that supports bazel persistent worker protocol.
+ * 
+ * Bazel can spawn a persistent worker process that handles multiple invocations. It
+ * will also be backwards compatible with being run as a normal single-invocation command.
+ */
+const {debug, log, runAsWorker, runWorkerLoop} = require('./worker');
+const sass = require('sass');
+const fs = require('fs');
+
+// Equivalent of running node with --expose-gc
+// but easier to write tooling since we don't need to inject that arg to
+// nodejs_binary
```

Comment:
Can this comment explain in more detail why we'd want this to be exposed? It looks like nothing is referring to it in practice.

Reply:
It was not needed, but with my most recent change that inlines the worker.js into the same file, it is now needed.

Reply:
I'd still like this comment to explain *why* it's needed.

Reply:
It was removed (I mean to delete this anyway), so no comment is needed :)